Makefile

```diff
@@ -14,10 +14,9 @@ TEST_REPORT_FILE := nose2-junit.xml
 
 # general targets timestamps
 TIMESTAMPS = .timestamps
-REQUIREMENTS_TIMESTAMP = $(TIMESTAMPS)/.requirements.timestamp
-DEV_REQUIREMENTS_TIMESTAMP = $(TIMESTAMPS)/.dev-requirements.timestamps
+REQUIREMENTS_TIMESTAMP = $(LOGS_DIR) $(TIMESTAMPS)/.requirements.timestamp
+DEV_REQUIREMENTS_TIMESTAMP = $(LOGS_DIR) $(TIMESTAMPS)/.dev-requirements.timestamps
```

Comment:
`$(LOGS_DIR)` not required here it is already present below in the target dependencies. Those lines should stay unchanged.